zubhub_backend/zubhub/creators/serializers.py

```diff
@@ -198,8 +199,6 @@ class CustomRegisterSerializer(RegisterSerializer):
                                             queryset=Location.objects.all())
     bio = serializers.CharField(allow_blank=True, default="", max_length=255)
     subscribe = serializers.BooleanField(default=False)
-    creator_tags = serializers.SlugRelatedField(slug_field='name',
```

Comment:
the syntax here is basically saying `Creator.objects.all().filter(name=<value of creator_tags coming from the frontend>)[0], or something like that`. But the value of the creator_tags coming from the frontend given the way we are trying to extract it and the default we are providing, is an array, so won't work